openedx/core/djangoapps/content/course_overviews/tests/test_course_overviews.py

```diff
@@ -355,11 +355,10 @@ def test_course_overview_saving_race_condition(self):
                 'openedx.core.djangoapps.content.course_overviews.models.CourseOverview._get_pk_val'
             ) as mock_get_pk_val:
                 mock_get_pk_val.return_value = None
-                # This method was not present in django 1.4. Django 1.8 calls this method if
-                # _get_pk_val returns None.  This method will return empty str if there is no
-                # default value present. So mock it to avoid returning the empty str as primary key
-                # value. Due to empty str, model.save will do an update instead of insert which is
-                # incorrect and get exception in
+                # Django 1.8+ calls this method if _get_pk_val returns None.  This method will
+                # return empty str if there is no default value present. So mock it to avoid
+                # returning the empty str as primary key value. Due to empty str, model.save will do
+                # an update instead of insert which is incorrect and get exception in
```

Comment:
Strikes me that there should be a better way to fix this. Could we backlog a ticket to do so please?

Reply:
It does cross some wires which seem very deep in the (ever-changing) internals of django.  Will backlog it.

Reply:
see PLAT-1757